### PR TITLE
4 get /api/articles

### PR DESCRIPTION
**Pull request by jaketysome**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -28,17 +28,51 @@ describe("api", () => {
...
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((response) => {
+          const articles = response.body.articles;
+          expect(articles).toBeInstanceOf(Array);
+          expect(articles).toHaveLength(12);
+          articles.forEach((article) => {
+            expect(article).toEqual(
+              expect.objectContaining({
```
Comment (B7Bprog) [inline __tests__/app.test.js:55]: Articles now should have a 'comment_count' property. Please include it in your test and send us a quick pull request with it.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -28,17 +28,51 @@ describe("api", () => {
...
+    });
+  });
+  describe("/api/articles", () => {
+    test("status: 200, should return an array of article objects", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((response) => {
+          const articles = response.body.articles;
+          expect(articles).toBeInstanceOf(Array);
```
Comment (B7Bprog) [inline __tests__/app.test.js]: Is this necessary to check even you are using .forEach() and it only works with arrays?

**Review by B7Bprog — CHANGES_REQUESTED** [review]
Doing well Jake, please see my comments, make the recommended changes and resubmit a pr. 👍

File: __tests__/app.test.js (hunk 1 above)
Comment (jaketysome) [reply __tests__/app.test.js:55]: I'm not sure where this comment_count would be coming from?

File: __tests__/app.test.js (hunk 2 above)
Comment (jaketysome) [reply __tests__/app.test.js]: ok, yes understood!

File: __tests__/app.test.js (hunk 1 above)
Comment (jaketysome) [reply __tests__/app.test.js:55]: Ignore me actually - I understand where I have gone wrong here - thank you!

**Pull request closed without merging**